options.go

```diff
@@ -84,3 +87,14 @@ func OnEvicted(fn func(key string, value Value)) Option {
 		return nil
 	}
 }
+
+// RedisOptions functional option defines duration.
+// Works for RedisCache only
+func RedisOptions(Addr string, Password string, DB int) Option {
```

Comment:
redis params are not common options and not optional values at all. I'd suggest moving it as a struct param to redis backend constructor